scripts/zones/Cape_Teriggan/mobs/Zmey_Gorynych.lua

```diff
@@ -4,8 +4,21 @@
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    -- This is a WAR mob, but does not use any double attack.
```

Comment:
All 3 of jp wiki bg and ffxiclopedia say its thf, jp wiki even claims thf+war with _both_ double and triple attack

> [攻撃力](https://wiki.ffo.jp/html/1766.html)が元々高い上に、運が悪いと[DA](https://wiki.ffo.jp/html/3922.html)＆[TA](https://wiki.ffo.jp/html/7800.html)による連続[攻撃](https://wiki.ffo.jp/html/11495.html)によって一瞬で[HP](https://wiki.ffo.jp/html/324.html)を削られることがある。[回避盾](https://wiki.ffo.jp/html/10075.html)が有効ではあるが、[攻撃](https://wiki.ffo.jp/html/11495.html)の[命中率](https://wiki.ffo.jp/html/226.html)も高く蝉は剥げやすい。[特殊技](https://wiki.ffo.jp/html/12837.html)は[サイズテール](https://wiki.ffo.jp/html/23463.html)しか使わず、終盤では怒涛のように使ってくる。[サイズテール](https://wiki.ffo.jp/html/23463.html)の直撃を受けると、[即死](https://wiki.ffo.jp/html/10818.html)の可能性が高く注意が必要。

which says 

> Not only does it have high attack power, but if you're unlucky, continuous attacks from DA & TA can instantly reduce its HP. Although the evasion shield is effective, the hit rate of attacks is high and the cicadas are easy to peel off. The only special technique he uses is Size Tail, and he uses it like a wave in the final stages. If you receive a direct hit from a size tail, there is a high possibility of instant death, so be careful.

Reply:
my point isn't that it has double (wiki's can be wrong!), its that its likely NOT a war

Reply:
so. I only just now noticed its already set thf/thf in pools and feel silly for not just telling you it doesn't have any DA to have to zero.